package.json

```diff
@@ -55,7 +55,7 @@
   },
   "devDependencies": {
     "@hapi/boom": "^9.0.0",
-    "@hapi/hapi": "^20.0.0",
+    "@hapi/hapi": "^20.2.1",
     "@vrbo/service-client": "^3.0.0",
```

Comment:
Before merge this we need to upgrade the `@vrbo/service-client` at the new release
for `#65` [service-client](https://github.com/ExpediaGroup/service-client/pull/65)

Reply:
Can you review which repos need to merge first?